src/kt_variant.h

```diff
@@ -236,12 +236,12 @@ namespace ktvariant {
         //TODO: Manage 32 bits systems.
         des->increment_position(encode_uint64(reinterpret_cast<uintptr_t>(ptr), des->get_cursor()));
 
-        String class_name {ptr->get_class()};
+        StringName class_name{ptr->get_class()};
 
         if (!TypeManager::get_instance().JAVA_ENGINE_TYPES_CONSTRUCTORS.has(class_name)) {
             class_name = ClassDB::get_parent_class(class_name);
 
-            while (class_name.empty()) {
+            while (class_name == String()) {
```

Comment:
@piiertho Is this still needed?

Reply:
As we now directly use a `StringName` yes.